mscore/editdrumset.cpp

```diff
@@ -102,14 +118,83 @@ EditDrumset::EditDrumset(const Drumset* ds, QWidget* parent)
       pitchList->setColumnWidth(1, 60);
       pitchList->setColumnWidth(2, 30);
 
+      QStringList validNoteheadRanges = { "Noteheads", "Slash noteheads", "Round and square noteheads", "Shape note noteheads", "Shape note noteheads supplement" };
+      QSet<QString> excludeSym = {"noteheadParenthesisLeft", "noteheadParenthesisRight"};
+      struct SymbolIcon {
+            SymId id;
+            QIcon icon;
+            SymbolIcon(SymId i, QIcon j)
+                  : id(i), icon(j)
+                  {}
+      };
+      int w = quarterCmb->iconSize().width()  * qApp->devicePixelRatio();
+      int h = quarterCmb->iconSize().height() * qApp->devicePixelRatio();
+      QList<SymbolIcon> validNoteheads;
+      for (QString range : validNoteheadRanges) {
+            for (auto symName : (*MuseScore::bravuraRanges())[range]) {
+                   SymId id = Sym::name2id(symName);
+                   if (!excludeSym.contains(symName)) {
+                         QIcon icon;
+                         QPixmap image(w, h);
+                         image.fill(Qt::transparent);
+                         QPainter painter(&image);
+                         const QRectF& bbox = ScoreFont::fallbackFont()->bbox(id, 1);
+                         //default scale is 1.3, use smaller scale for large noteheads symbols
+                         qreal mag = std::min(1.3, std::min(w / bbox.width(), h / bbox.height()));
```

Comment:
Maybe it's my retina display but 2.6 looks a lot better on my mac. Same for master btw.
Maybe we should use *qApp->devicePixelRatio()

Reply:
Thanks, will experiment with it.